### ISSUE TITLE
PytestUnknownMarkWarning on pytest:5 + pytest-xdist

### ISSUE THREAD

**Issue by spumer** [renanivo/pytest-testdox#31]
When i run `pytest --dead-fixtures --dup-fixtures` i got many warnings

```
/Users/posokhinsvyatoslav/.pyenv/versions/3.7.3/envs/hr-one-venv/lib/python3.7/site-packages/_pytest/mark/structures.py:332: PytestUnknownMarkWarning: Unknown pytest.mark.describe - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
  PytestUnknownMarkWarning,
```

I got it after install `pytest-xdist` and upgrade `pytest-testdox` to 1.2.0. On version 1.1.1 this checks crashed.

**requirements-test.txt**

```
factory_boy==2.11.1
flake8-awesome==1.2.0
freezegun==0.3.12
pytest==5.0.1
pytest-cov==2.7.1
pytest-deadfixtures==2.1.0
pytest-django==3.5.1
pytest-env==0.6.2
pytest-lazy-fixture==0.6.1
pytest-testdox==1.2.0
pytest-xdist==1.30.0
pytest_mock==1.10.0
requests_mock==1.7.0
unify==0.5
```

Can you enable marks registration in any way?
https://github.com/renanivo/pytest-testdox/blob/1.2.0/pytest_testdox/plugin.py#L47

**Comment by renanivo** [comment]
Got it. Seems to be easy to fix. Thank you for the report. I think I can do it in the next couple weeks, but if anyone is reading this message and would like to do it, just leave a comment.

**Comment by netolyrg** [comment]
@renanivo hi! 
I think I fixed it :)

**Comment by renanivo** [comment]
Fixed at #32
